scripts/cronjob.js

```diff
@@ -0,0 +1,5 @@
+const CronJob = require("cron").CronJob
+
+new CronJob("* * * * * *", function() {
+  console.log("You will see this message every second")
```

Comment:
Since you need to run truffle run it might be easier to use the unix crontab (if you manage to get that to work)

Reply:
Yes, this was just an experiment